src/Symfony/Component/RemoteEvent/Event/Sms/SmsEvent.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\RemoteEvent\Event\Sms;
+
+use Symfony\Component\RemoteEvent\Event;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 6.3
+ */
+final class SmsEvent extends Event
+{
+    public const FAILED = 'failed';
+    public const DELIVERED = 'delivered';
+
+    private string $sms = '';
```

Comment:
I'm having some trouble understanding which is this `$sms` property. Is it the SMS content or the SMS recipient phone number? The method is called `get/setRecipientSms()` so I guess this variable stores a phone number, but I'm not sure because of its name.

Reply:
Renamed to `phone` as this is what we use in `SmsRecipientInterface` Notifier.